diff -Naur nghttp2-1.4.0/configure nghttp2-1.4.0.oden/configure
--- nghttp2-1.4.0/configure	2015-10-25 09:32:29.000000000 +0100
+++ nghttp2-1.4.0.oden/configure	2015-11-05 16:07:05.226854798 +0100
@@ -18277,7 +18277,7 @@
 fi
 
 if test "x${have_libev}" = "xyes"; then
-  ac_fn_c_check_header_mongrel "$LINENO" "ev.h" "ac_cv_header_ev_h" "$ac_includes_default"
+  ac_fn_c_check_header_mongrel "$LINENO" "libev/ev.h" "ac_cv_header_ev_h" "$ac_includes_default"
 if test "x$ac_cv_header_ev_h" = xyes; then :
   have_libev=yes
 else
diff -Naur nghttp2-1.4.0/configure.ac nghttp2-1.4.0.oden/configure.ac
--- nghttp2-1.4.0/configure.ac	2015-10-25 09:32:14.000000000 +0100
+++ nghttp2-1.4.0.oden/configure.ac	2015-11-05 16:07:05.226854798 +0100
@@ -278,7 +278,7 @@
 # android requires -lm for floor
 AC_CHECK_LIB([ev], [ev_time], [have_libev=yes], [have_libev=no], [-lm])
 if test "x${have_libev}" = "xyes"; then
-  AC_CHECK_HEADER([ev.h], [have_libev=yes], [have_libev=no])
+  AC_CHECK_HEADER([libev/ev.h], [have_libev=yes], [have_libev=no])
   if test "x${have_libev}" = "xyes"; then
     LIBEV_LIBS=-lev
     LIBEV_CFLAGS=
diff -Naur nghttp2-1.4.0/src/h2load.h nghttp2-1.4.0.oden/src/h2load.h
--- nghttp2-1.4.0/src/h2load.h	2015-10-25 09:32:14.000000000 +0100
+++ nghttp2-1.4.0.oden/src/h2load.h	2015-11-05 16:08:44.405622045 +0100
@@ -44,7 +44,7 @@
 
 #include <nghttp2/nghttp2.h>
 
-#include <ev.h>
+#include <libev/ev.h>
 
 #include <openssl/ssl.h>
 
diff -Naur nghttp2-1.4.0/src/HttpServer.h nghttp2-1.4.0.oden/src/HttpServer.h
--- nghttp2-1.4.0/src/HttpServer.h	2015-10-25 09:32:14.000000000 +0100
+++ nghttp2-1.4.0.oden/src/HttpServer.h	2015-11-05 16:08:44.408622007 +0100
@@ -39,7 +39,7 @@
 
 #include <openssl/ssl.h>
 
-#include <ev.h>
+#include <libev/ev.h>
 
 #include <nghttp2/nghttp2.h>
 
diff -Naur nghttp2-1.4.0/src/nghttp.h nghttp2-1.4.0.oden/src/nghttp.h
--- nghttp2-1.4.0/src/nghttp.h	2015-10-25 09:32:14.000000000 +0100
+++ nghttp2-1.4.0.oden/src/nghttp.h	2015-11-05 16:08:44.411621970 +0100
@@ -43,7 +43,7 @@
 
 #include <openssl/ssl.h>
 
-#include <ev.h>
+#include <libev/ev.h>
 
 #include <nghttp2/nghttp2.h>
 
diff -Naur nghttp2-1.4.0/src/shrpx_accept_handler.h nghttp2-1.4.0.oden/src/shrpx_accept_handler.h
--- nghttp2-1.4.0/src/shrpx_accept_handler.h	2015-10-25 09:32:14.000000000 +0100
+++ nghttp2-1.4.0.oden/src/shrpx_accept_handler.h	2015-11-05 16:08:44.411621970 +0100
@@ -27,7 +27,7 @@
 
 #include "shrpx.h"
 
-#include <ev.h>
+#include <libev/ev.h>
 
 namespace shrpx {
 
diff -Naur nghttp2-1.4.0/src/shrpx.cc nghttp2-1.4.0.oden/src/shrpx.cc
--- nghttp2-1.4.0/src/shrpx.cc	2015-10-25 09:32:14.000000000 +0100
+++ nghttp2-1.4.0.oden/src/shrpx.cc	2015-11-05 16:08:44.412621958 +0100
@@ -67,7 +67,7 @@
 
 #include <openssl/ssl.h>
 #include <openssl/err.h>
-#include <ev.h>
+#include <libev/ev.h>
 
 #include <nghttp2/nghttp2.h>
 
diff -Naur nghttp2-1.4.0/src/shrpx_client_handler.h nghttp2-1.4.0.oden/src/shrpx_client_handler.h
--- nghttp2-1.4.0/src/shrpx_client_handler.h	2015-10-25 09:32:14.000000000 +0100
+++ nghttp2-1.4.0.oden/src/shrpx_client_handler.h	2015-11-05 16:08:44.412621958 +0100
@@ -29,7 +29,7 @@
 
 #include <memory>
 
-#include <ev.h>
+#include <libev/ev.h>
 
 #include <openssl/ssl.h>
 
diff -Naur nghttp2-1.4.0/src/shrpx_config.h nghttp2-1.4.0.oden/src/shrpx_config.h
--- nghttp2-1.4.0/src/shrpx_config.h	2015-10-25 09:32:14.000000000 +0100
+++ nghttp2-1.4.0.oden/src/shrpx_config.h	2015-11-05 16:08:44.413621945 +0100
@@ -46,7 +46,7 @@
 
 #include <openssl/ssl.h>
 
-#include <ev.h>
+#include <libev/ev.h>
 
 #include <nghttp2/nghttp2.h>
 
diff -Naur nghttp2-1.4.0/src/shrpx_connect_blocker.h nghttp2-1.4.0.oden/src/shrpx_connect_blocker.h
--- nghttp2-1.4.0/src/shrpx_connect_blocker.h	2015-10-25 09:32:14.000000000 +0100
+++ nghttp2-1.4.0.oden/src/shrpx_connect_blocker.h	2015-11-05 16:08:44.413621945 +0100
@@ -27,7 +27,7 @@
 
 #include "shrpx.h"
 
-#include <ev.h>
+#include <libev/ev.h>
 
 namespace shrpx {
 
diff -Naur nghttp2-1.4.0/src/shrpx_connection.h nghttp2-1.4.0.oden/src/shrpx_connection.h
--- nghttp2-1.4.0/src/shrpx_connection.h	2015-10-25 09:32:14.000000000 +0100
+++ nghttp2-1.4.0.oden/src/shrpx_connection.h	2015-11-05 16:08:44.414621933 +0100
@@ -29,7 +29,7 @@
 
 #include <sys/uio.h>
 
-#include <ev.h>
+#include <libev/ev.h>
 
 #include <openssl/ssl.h>
 
diff -Naur nghttp2-1.4.0/src/shrpx_connection_handler.h nghttp2-1.4.0.oden/src/shrpx_connection_handler.h
--- nghttp2-1.4.0/src/shrpx_connection_handler.h	2015-10-25 09:32:14.000000000 +0100
+++ nghttp2-1.4.0.oden/src/shrpx_connection_handler.h	2015-11-05 16:08:44.414621933 +0100
@@ -40,7 +40,7 @@
 
 #include <openssl/ssl.h>
 
-#include <ev.h>
+#include <libev/ev.h>
 
 #ifdef HAVE_NEVERBLEED
 #include <neverbleed.h>
diff -Naur nghttp2-1.4.0/src/shrpx_downstream.h nghttp2-1.4.0.oden/src/shrpx_downstream.h
--- nghttp2-1.4.0/src/shrpx_downstream.h	2015-10-25 09:32:14.000000000 +0100
+++ nghttp2-1.4.0.oden/src/shrpx_downstream.h	2015-11-05 16:08:44.415621920 +0100
@@ -33,7 +33,7 @@
 #include <memory>
 #include <chrono>
 
-#include <ev.h>
+#include <libev/ev.h>
 
 #include <nghttp2/nghttp2.h>
 
diff -Naur nghttp2-1.4.0/src/shrpx_http2_session.h nghttp2-1.4.0.oden/src/shrpx_http2_session.h
--- nghttp2-1.4.0/src/shrpx_http2_session.h	2015-10-25 09:32:14.000000000 +0100
+++ nghttp2-1.4.0.oden/src/shrpx_http2_session.h	2015-11-05 16:08:44.416621908 +0100
@@ -32,7 +32,7 @@
 
 #include <openssl/ssl.h>
 
-#include <ev.h>
+#include <libev/ev.h>
 
 #include <nghttp2/nghttp2.h>
 
diff -Naur nghttp2-1.4.0/src/shrpx_http2_upstream.h nghttp2-1.4.0.oden/src/shrpx_http2_upstream.h
--- nghttp2-1.4.0/src/shrpx_http2_upstream.h	2015-10-25 09:32:14.000000000 +0100
+++ nghttp2-1.4.0.oden/src/shrpx_http2_upstream.h	2015-11-05 16:08:44.417621895 +0100
@@ -29,7 +29,7 @@
 
 #include <memory>
 
-#include <ev.h>
+#include <libev/ev.h>
 
 #include <nghttp2/nghttp2.h>
 
diff -Naur nghttp2-1.4.0/src/shrpx_io_control.h nghttp2-1.4.0.oden/src/shrpx_io_control.h
--- nghttp2-1.4.0/src/shrpx_io_control.h	2015-10-25 09:32:14.000000000 +0100
+++ nghttp2-1.4.0.oden/src/shrpx_io_control.h	2015-11-05 16:08:44.418621883 +0100
@@ -29,7 +29,7 @@
 
 #include <vector>
 
-#include <ev.h>
+#include <libev/ev.h>
 
 #include "shrpx_rate_limit.h"
 
diff -Naur nghttp2-1.4.0/src/shrpx_memcached_connection.h nghttp2-1.4.0.oden/src/shrpx_memcached_connection.h
--- nghttp2-1.4.0/src/shrpx_memcached_connection.h	2015-10-25 09:32:14.000000000 +0100
+++ nghttp2-1.4.0.oden/src/shrpx_memcached_connection.h	2015-11-05 16:08:44.418621883 +0100
@@ -30,7 +30,7 @@
 #include <memory>
 #include <deque>
 
-#include <ev.h>
+#include <libev/ev.h>
 
 #include "shrpx_connection.h"
 #include "buffer.h"
diff -Naur nghttp2-1.4.0/src/shrpx_memcached_dispatcher.h nghttp2-1.4.0.oden/src/shrpx_memcached_dispatcher.h
--- nghttp2-1.4.0/src/shrpx_memcached_dispatcher.h	2015-10-25 09:32:14.000000000 +0100
+++ nghttp2-1.4.0.oden/src/shrpx_memcached_dispatcher.h	2015-11-05 16:08:44.418621883 +0100
@@ -29,7 +29,7 @@
 
 #include <memory>
 
-#include <ev.h>
+#include <libev/ev.h>
 
 namespace shrpx {
 
diff -Naur nghttp2-1.4.0/src/shrpx_rate_limit.h nghttp2-1.4.0.oden/src/shrpx_rate_limit.h
--- nghttp2-1.4.0/src/shrpx_rate_limit.h	2015-10-25 09:32:14.000000000 +0100
+++ nghttp2-1.4.0.oden/src/shrpx_rate_limit.h	2015-11-05 16:08:44.419621871 +0100
@@ -27,7 +27,7 @@
 
 #include "shrpx.h"
 
-#include <ev.h>
+#include <libev/ev.h>
 
 #include <openssl/ssl.h>
 
diff -Naur nghttp2-1.4.0/src/shrpx_spdy_upstream.h nghttp2-1.4.0.oden/src/shrpx_spdy_upstream.h
--- nghttp2-1.4.0/src/shrpx_spdy_upstream.h	2015-10-25 09:32:14.000000000 +0100
+++ nghttp2-1.4.0.oden/src/shrpx_spdy_upstream.h	2015-11-05 16:08:44.420621858 +0100
@@ -29,7 +29,7 @@
 
 #include <memory>
 
-#include <ev.h>
+#include <libev/ev.h>
 
 #include <spdylay/spdylay.h>
 
diff -Naur nghttp2-1.4.0/src/shrpx_ssl.h nghttp2-1.4.0.oden/src/shrpx_ssl.h
--- nghttp2-1.4.0/src/shrpx_ssl.h	2015-10-25 09:32:14.000000000 +0100
+++ nghttp2-1.4.0.oden/src/shrpx_ssl.h	2015-11-05 16:08:44.421621846 +0100
@@ -33,7 +33,7 @@
 #include <openssl/ssl.h>
 #include <openssl/err.h>
 
-#include <ev.h>
+#include <libev/ev.h>
 
 #ifdef HAVE_NEVERBLEED
 #include <neverbleed.h>
diff -Naur nghttp2-1.4.0/src/shrpx_worker.h nghttp2-1.4.0.oden/src/shrpx_worker.h
--- nghttp2-1.4.0/src/shrpx_worker.h	2015-10-25 09:32:14.000000000 +0100
+++ nghttp2-1.4.0.oden/src/shrpx_worker.h	2015-11-05 16:08:44.422621833 +0100
@@ -37,7 +37,7 @@
 #include <openssl/ssl.h>
 #include <openssl/err.h>
 
-#include <ev.h>
+#include <libev/ev.h>
 
 #include "shrpx_config.h"
 #include "shrpx_downstream_connection_pool.h"
diff -Naur nghttp2-1.4.0/src/shrpx_worker_process.cc nghttp2-1.4.0.oden/src/shrpx_worker_process.cc
--- nghttp2-1.4.0/src/shrpx_worker_process.cc	2015-10-25 09:32:14.000000000 +0100
+++ nghttp2-1.4.0.oden/src/shrpx_worker_process.cc	2015-11-05 16:08:44.422621833 +0100
@@ -36,7 +36,7 @@
 
 #include <openssl/rand.h>
 
-#include <ev.h>
+#include <libev/ev.h>
 
 #include "shrpx_config.h"
 #include "shrpx_connection_handler.h"
